src/core/core.scale.js

```diff
@@ -709,6 +709,12 @@ module.exports = Element.extend({
 		var xTickEnd = options.position === 'right' ? me.left + tl : me.right;
 		var yTickStart = options.position === 'bottom' ? me.top + axisWidth : me.bottom - tl - axisWidth;
 		var yTickEnd = options.position === 'bottom' ? me.top + axisWidth + tl : me.bottom + axisWidth;
+		var yRotationOffset = 0;
+
+		if (me.labelRotation < 0) {
```

Comment:
This isn't right. What if the labelRotation is 270 degrees. We should make sure it works in that case as well. I think what you want is to check `if (Math.sin(labelRotationRadians) < 0)` ?